apps/bmd/src/utils/Card.ts

```diff
@@ -159,8 +159,7 @@ export class MemoryCard implements Card {
    */
   public async readLongObject<T>(): Promise<Optional<T>> {
     const { longValue } = this
-
-    if (!longValue) {
+    if (!longValue || longValue.length === 0) {
```

Comment:
What triggered this change? Should we add a test for it?

Reply:
I suspect it's that we are clearing the card data on the poll worker card... and in the case of the voter card, where this also happens, the case has never come up because an expired voter card doesn't lead to reading the long value.

Reply:
I was getting crazy explosions in integration tests before updating this (this is in MemoryCard, so this just impacts the test environment ) after clearing the long value, longValue here would be an empty UInt8Array and when you pass that to the TextDecoder below it exploded.